lib/yabeda/active_record.rb

```diff
@@ -71,7 +71,12 @@ class Error < StandardError; end
 
       # Connection pool metrics collection
       collect do
-        connection_pools = ::ActiveRecord::Base.connection_handler.connection_pool_list
+        connection_pools = \
+          if Rails.version >= "7.1"
```

Comment:
```suggestion
          if ::ActiveRecord.version >= "7.1"
```

This gem doesn't depend on whole Rails and shouldn't expect its presence. It is possible to use ActiveRecord with Sinatra, for example.

Reply:
Whops, sorry! Thanks for the suggestion.